### PR TITLE
Wizard Components + Rollup Fixes

### PR DESCRIPTION
**Pull request by ColtenVanTussenbrook**
- Breaks up Wizard Header into Logo, Device Selection and Exit Button components
- Removes Navigation component and references to it
- Generate sourcemap in rollup, remove unneeded warnings

### CODE REVIEW

**Review by chumpgrub — APPROVED** [review]
I like the refactoring, here! I had one note/question.

File: src/components/logo/logo.tsx [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+import { styled } from '@mui/material/styles';
+
+interface LogoContainerProps extends BoxProps {
+  logoSrc?: string;
+  logoAlt?: string;
+  width?: string;
+}
+
+const LogoContainer = styled(Box, {
+  name: 'WmeWizardHeader',
```
Comment (alexdelgado) [inline src/components/logo/logo.tsx:15]: Should this be `WmeLogoContainer` or something similar?

**Review by alexdelgado — APPROVED** [review]

File: src/components/logo/logo.tsx (hunk 1 above)
Comment (ColtenVanTussenbrook) [reply src/components/logo/logo.tsx:15]: yes good catch, copy/paste error

File: src/components/device-selection/device-selection.tsx [hunk 2]
```diff
@@ -0,0 +1,57 @@
+import React, { ReactElement, ReactNode } from 'react';
```
[the remark this answers is not in the document]
Comment (ColtenVanTussenbrook) [reply src/components/device-selection/device-selection.tsx:1]: good catch

**Pull request merged**